Assignment2/q080099.ttl

```diff
@@ -0,0 +1,14 @@
+@base <http://www.oeg-upm.net/alumnos/asignment>
+@prefix vocab: <http://www.oeg-upm.net/alumnos/asignment#>
+:Class01 vocab:includes: Sensor029;
+vocab:includes: Computer101
+.
+:Sensor029 vocab:hasMeasurement: Measurement8401
+.
+:Computer101 vocab:hasOwner: User10A
+.
+:Measurement8401 vocab:hasTemperature "29";
```

Comment:
It is important to specify the datatypes for these literals. Same for the following line.